src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfStyles = sourceString.split(':').join(';').split(';').map(item =>
+    item.trim());
+  const css = {};
+
+  for (let i = 0; i < arrayOfStyles.length; i++) {
+    if (arrayOfStyles[i] !== '') {
+      css[arrayOfStyles[i]] = arrayOfStyles[i + 1];
+      i++;
+    }
```

Comment:
let's create array of arrays in the first place, so you don't need to use indexes that much

let's try to create such array out of the initial string
`[[property, value], [property, value], etc]`